SUMMARY.md

```diff
@@ -481,31 +489,26 @@
 - [Hardware Development Stages](./doc/project_governance/development_stages.md)
 - [Signoff Checklist](./doc/project_governance/checklist/README.md)
 
-# Security
-
-- [Security](./doc/security/README.md)
-- [Implementation Guidelines](./doc/security/implementation_guidelines/README.md)
-  - [Secure Hardware Design Guidelines](./doc/security/implementation_guidelines/hardware/README.md)
-
-- [Logical Security Model](./doc/security/logical_security_model/README.md)
-- [Security Model Specification](./doc/security/specs/README.md)
-  - [Device Attestation](./doc/security/specs/attestation/README.md)
-  - [Device Life Cycle](./doc/security/specs/device_life_cycle/README.md)
-  - [Device Provisioning](./doc/security/specs/device_provisioning/README.md)
-  - [Firmware Update](./doc/security/specs/firmware_update/README.md)
-  - [Identities and Root Keys](./doc/security/specs/identities_and_root_keys/README.md)
-  - [Ownership Transfer](./doc/security/specs/ownership_transfer/README.md)
-  - [Secure Boot](./doc/security/specs/secure_boot/README.md)
-
-- [Lightweight Threat Model](./doc/security/threat_model/README.md)
-
-# Use Cases
-
-- [Use Cases](./doc/use_cases/README.md)
-- [Platform Integrity Module](./doc/use_cases/platform_integrity_module/README.md)
-- [Trusted Platform Module](./doc/use_cases/tpm/README.md)
-- [Universal 2nd-Factor Security Key](./doc/use_cases/u2f/README.md)
+# Tooling
 
+- [Tools Overview](./util/README.md)
+- [Design-Related Tooling](./util/design/README.md)
+  - [topgen: Top Generator](./util/topgen/README.md)
+    - [tlgen: Crossbar Generation](./util/tlgen/README.md)
+    - [ipgen: Generate IP Blocks from IP Templates](./util/ipgen/README.md)
+    - [reggen & regtool: Register Generator](./util/reggen/README.md)
+      - [Setup and use of regtool](./util/reggen/doc/setup_and_use.md)
+- [DV-Related Tooling]()
```

Comment:
We should give all summary sections a page - mdbook considers these pages drafts and makes the titles hard to read:

![image](https://github.com/lowRISC/opentitan/assets/105280833/99980b46-331c-41f8-bcb8-3927c98fb6e2)

Alternatively we update the mdbook theme to make these more readable.

Reply:
It doesn't feel great to add pages just to get around a theme issue. Especially if we just put boilerplate text in there, it seems more distracting than useful IMO. But I agree the un-clickable, faded titles also are not great UI. 
Thinking about it a bit more, it would be nice for the title to just opens the subsections when clicked.